### PR TITLE
Update UtilityNetwork README and main README file

### PR DESCRIPTION
**Pull request by puneet-pdx**
Related to issue: # https://devtopia.esri.com/runtime/kotlin/issues/4912

### Description:

Updates the content of the readme files

### Summary of changes:

- Add screenshot of the Trace app
- Update the UtilityNetwork README file
- Update the project's readme file
- Add README for the micro app

### Pre-merge Checklist

- a [vTest](https://runtime-kotlin.esri.com/view/all/job/vtest/job/toolkit/) Job for this PR has been run
  - [x] link: https://runtime-kotlin.esri.com/view/all/job/vtest/job/toolkit/193/
- Unit and/or integration tests have been added to exercise this PR's logic, and the tests are passing:
  - [ ] Yes
  - [ ] No

### CODE REVIEW

**Review by TADraeseke — APPROVED** [review]
LGTM!

**Review by sorenoid — CHANGES_REQUESTED** [review]
Can you please add a README for the trace tool microapp? It is the only microapp without a README.

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ repositories {
 The *ArcGIS Maps SDK for Kotlin Toolkit* is released with a "bill of materials" (`BOM`). The releasable BOM is versioned and represents a set of versions of the toolkit components which are compatible with one another. You may specify dependencies as follows
 
 ```
-implementation(platform("com.esri:arcgis-maps-kotlin-toolkit-bom:200.5.0"))
+implementation(platform("com.esri:arcgis-maps-kotlin-toolkit-bom:200.6.0"))
```
Comment (sorenoid) [inline README.md:47]:
```suggestion
implementation(platform("com.esri:arcgis-maps-kotlin-toolkit-bom:200.6.0"))
implementation("com.esri:arcgis-maps-kotlin-toolkit-ar")
```

Comment (sorenoid) [reply README.md:47]: The ar component now needs to be added here...

**Review by sorenoid — CHANGES_REQUESTED** [review]
just one small change. Please also consider the text that lists the styles used in the trace tool for its README.

File: README.md (hunk 1 above)
Comment (puneet-pdx) [reply README.md:47]: Not needed. Gunther merged changes to this file in his PR https://github.com/Esri/arcgis-maps-sdk-kotlin-toolkit/pull/664/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R56

**Review by sorenoid — APPROVED** [review]

**Pull request merged**